app/services/manage_braintree_donation.rb

```diff
@@ -62,6 +62,25 @@ def user_params
     )
   end
 
+  # ActionKit can accept one of the following:
+  #
+  # PayPal USD
+  # PayPal GBP
+  # PayPal CAD
+  # PayPal EUR
+  # PayPal AUD
+  #
+  # Braintree USD
+  # Braintree CAD
+  # Braintree AUD
+  # Braintree GBP
+  # Braintree EUR
+  #
+  def get_payment_account
+    provider = card_num == PAYPAL_IDENTIFIER ? 'PayPal' : 'Braintree'
```

Comment:
this references as `ManageBraintreeDonation::PAYPAL_IDENTIFIER` and `PAYPAL_IDENTIFIER`, can it be consistent?